### PR TITLE
Destroy contract

### PR DESCRIPTION
**Pull request by joydark**

### CODE REVIEW

File: __tests__/contracts.js [hunk 1]
```diff
@@ -999,3 +1000,48 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
...
+
+    const originalBalance = (await queries.accounts.waitFor(
+        {
+            id: { eq: contractData.address },
+        },
+        'balance',
+    )).balance;
+
+    // eslint-disable-next-line no-bitwise
+    const flags = 0 | 128;
```
Comment (AlexeyVavilin) [inline __tests__/contracts.js:1022]: Why not `const flags = 128;`?

File: __tests__/contracts.js [hunk 2]
```diff
@@ -999,3 +1000,48 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
     expect(checked.alreadyDeployed)
         .toBeTruthy();
 });
+test('do_tvm_transfer() should carry all the remaining balance of the current smart contract to another address', async () => {
+    const { contracts, crypto, queries } = tests.client;
+    const giverKeys = await crypto.ed25519Keypair();
+
+    const transferPackage = TransferContractPackage[2];
```
Comment (AlexeyVavilin) [inline __tests__/contracts.js:1007]: Only ABI v2? But you added contract to abi_v1 folder too. Why not using them?

Comment (joydark) [reply __tests__/contracts.js]: fixed

File: __tests__/contracts.js (hunk 1 above)
Comment (joydark) [reply __tests__/contracts.js]: Revert to 128 till fix for 0 |  32 | 128 test

**Pull request closed without merging**